nvme.c

```diff
@@ -5273,14 +5361,18 @@ static int nvme_get_properties(int fd, void **pbar)
 	return err;
 }
 
-static void *mmap_registers(nvme_root_t r, struct nvme_dev *dev)
+static void *mmap_registers(nvme_root_t r, struct nvme_dev *dev, bool writable)
```

Comment:
This really feels like it could be its own patch. It's refactoring code and preparing the stage for the get/set register feature.